tests/ha/single_pod_restart/test_data_pod_restart.py

```diff
@@ -713,11 +737,15 @@ def test_mpu_after_pod_restart(self):
                                   f"Failed to match checksum: {upload_checksum},"
                                   f" {download_checksum}")
         LOGGER.info("Matched checksum: %s, %s", upload_checksum, download_checksum)
-        LOGGER.info("Step 10: Successfully downloaded the object %s & verified the checksum",
+        LOGGER.info("Step 8: Successfully downloaded the object %s & verified the checksum",
                     test_object)
-        LOGGER.info("Removing files %s and %s", self.multipart_obj_path, download_path)
+        LOGGER.info("Removing files %s, %s, %s, %s", self.multipart_obj_path, download_path,
+                    download_path_1, download_path_2)
         system_utils.remove_file(self.multipart_obj_path)
         system_utils.remove_file(download_path)
+        system_utils.remove_file(download_path_1)
```

Comment:
we had remove_file in teardown isn't it?

Reply:
In teardown we are deleting files defined with self. But local files, we will have to remove them here.
system_utils.remove_file(self.multipart_obj_path) this i will move to teardown.

Reply:
Can we add all these files in a list instead and then remove in teardown?